From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Sun, 28 Aug 2022 12:06:37 -0400
Subject: Switch to webkitgtk 4.1

4.1 is the same as 4.0 except that it uses libsoup3 instead of libsoup2.4
---
 config.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/config.mk b/config.mk
index c98442b..0141b6f 100644
--- a/config.mk
+++ b/config.mk
@@ -13,10 +13,10 @@ LIBDIR = $(LIBPREFIX)/surf
 X11INC = $(shell $(PKG_CONFIG) --cflags x11)
 X11LIB = $(shell $(PKG_CONFIG) --libs x11)
 
-GTKINC = $(shell $(PKG_CONFIG) --cflags gtk+-3.0 gcr-3 webkit2gtk-4.0)
-GTKLIB = $(shell $(PKG_CONFIG) --libs gtk+-3.0 gcr-3 webkit2gtk-4.0)
-WEBEXTINC = $(shell $(PKG_CONFIG) --cflags webkit2gtk-4.0 webkit2gtk-web-extension-4.0 gio-2.0)
-WEBEXTLIBS = $(shell $(PKG_CONFIG) --libs webkit2gtk-4.0 webkit2gtk-web-extension-4.0 gio-2.0)
+GTKINC = $(shell $(PKG_CONFIG) --cflags gtk+-3.0 gcr-3 webkit2gtk-4.1)
+GTKLIB = $(shell $(PKG_CONFIG) --libs gtk+-3.0 gcr-3 webkit2gtk-4.1)
+WEBEXTINC = $(shell $(PKG_CONFIG) --cflags webkit2gtk-4.1 webkit2gtk-web-extension-4.1 gio-2.0)
+WEBEXTLIBS = $(shell $(PKG_CONFIG) --libs webkit2gtk-4.1 webkit2gtk-web-extension-4.1 gio-2.0)
 
 # includes and libs
 INCS = $(X11INC) $(GTKINC)
